index.bs

```diff
@@ -350,9 +354,15 @@ spec: webidl
   <p>
     Each enumeration value in the {{PermissionName}} enum identifies a
     <a>powerful feature</a>. Each <a>powerful feature</a> has the following
-    permission-related algorithms and types:
+    permission-related flags, algorithms, and types:
   </p>
   <dl>
+    <dt>A <dfn export>requires a secure context</dfn> flag</dt>
+    <dd>
+      Indicates that the feature will always be {{"denied"}} in <a>non-secure
+      contexts</a>. This flag defaults to un-set.
```

Comment:
Arguably, we should default this to "set", and make features explicitly say that they don't require a secure context... Then I'd probably reword it to reverse the sense: "allowed in non-secure contexts" defaulting again to un-set.

Thoughts?